system/jlib/jscm.hpp

```diff
@@ -81,6 +83,19 @@ template <class CLASS> class Shared
     }
     inline void set(const Shared<CLASS> &other) { this->set(other.get()); }
     inline void setown(CLASS * _ptr)            { CLASS * temp = ptr; ptr = _ptr; ::Release(temp); }
+    inline bool setownNull(CLASS * _ptr)
```

Comment:
Took me a few moments to process what this was for. Would setownIfNull be a better name?

Reply:
Or setownIfUnset

Reply:
yes, I think I originally thought of calling it setownIfNull().  A slightly unwieldy name is probably better than a confusing name.